addons/hello.php

```diff
@@ -3,6 +3,7 @@
 /* SIMPLE EXAMPLE HOW TO USE ADDON */
 
 // LANG
+global $l;
 $l['hello_title'] = 'Hello World';
```

Comment:
This is really strange but works.
Let explain: 
When you use one variable (not inside function) it will be global, and you don't need to declare as a global var.